### ISSUE TITLE
Error while changing information from "settings.general.json"

### ISSUE THREAD

**Issue by learts13** [Legedric/ptmagic#318]
Error while changing information from "settings.general.json"

![whatsapp image 2018-03-15 at 16 28 15](https://user-images.githubusercontent.com/36136476/37486985-f595090e-286e-11e8-8313-dccb8e53ec7c.jpeg)

The error occurs because when the "settings.general.json" file is saved the "TimezoneOffset": field receives the time format with a space between hour and minutes.

Here's how it is after first saving the file "settings.general.json" through the PTMagic interface:

![whatsapp image 2018-03-15 at 16 26 43](https://user-images.githubusercontent.com/36136476/37487140-68a08f90-286f-11e8-93a9-c8158dcae494.jpeg)

![whatsapp image 2018-03-15 at 16 25 21](https://user-images.githubusercontent.com/36136476/37487176-888fa476-286f-11e8-9ce0-eab3c28f0e2a.jpeg)

**Comment by Legedric** [comment]
Thanks, I will check this!

**Comment by learts13** [comment]
Já resolvi o problema. Já resolvi o problema. Já resolvi o problema. Já resolvi o problema. O erro acontecia Já resolvi o problema. Já resolvi o problema. Já resolvi o problema. Já resolvi o problema. O erro acontecia  Já resolvi o problema. Já resolvi o problema. Já resolvi o problema. Já resolvi o problema. O erro acontecia quando o C Já resolvi o problema. Já resolvi o problema. Já resolvi o problema. Já resolvi o problema. O erro acontecia quando o Ch I've solved the problem. I've solved the problem. I've solved the problem. I've solved the problem. The error happened when Chrome was in Portuguese language
